src/cljs/rems/application.cljs

```diff
@@ -925,17 +928,17 @@
 (defn- dynamic-actions [app]
   (distinct
    (mapcat #:rems.workflow.dynamic
-           {:submit [[save-button]
-                     [submit-button]]
-            :add-member nil ; TODO implement
-            :return [[return-action-button]]
-            :request-decision [[request-decision-action-button]]
-            :decide [[decide-action-button]]
-            :request-comment [[request-comment-action-button]]
-            :comment [[comment-action-button]]
-            :approve [[approve-reject-action-button]]
-            :reject [[approve-reject-action-button]]
-            :close [[close-action-button]]}
+               {:submit [[save-button]
```

Comment:
What is this whitespace change?

Reply:
It's how Cursive formats that code and there are no options for configuring it. There doesn't seem to be an established convention for formatting maps with namespaced keys - the syntax is not even mentioned at https://github.com/bbatsov/clojure-style-guide

Reply:
Looks quite strange option from cursive. I can reformat this later.